src/Store/VimStoreConnector.re

```diff
@@ -975,6 +980,42 @@ let start =
     });
   };
 
+  let zoomStep = 0.2;
+  let defaultZoom = 1.0;
+  let minZoomValue = 0.0;
+  let maxZoomValue = 2.8;
```

Comment:
Could you put these in a `Constants` sub-module at the top of the file?

Reply:
Yes, of course.